vocode/turn_based/synthesizer/google_synthesizer.py

```diff
@@ -0,0 +1,61 @@
+import io
+import os
+from typing import Optional
+from pydub import AudioSegment
+import requests
+from vocode import getenv
+from vocode.turn_based.synthesizer.base_synthesizer import BaseSynthesizer
+from google.cloud import texttospeech_v1beta1 as tts
+
+DEFAULT_GOOGLE_LANGUAGE_CODE = "en-US"
+DEFAULT_GOOGLE_VOICE_NAME = "en-US-Neural2-I"
+DEFAULT_GOOGLE_PITCH = 0
+DEFAULT_GOOGLE_SPEAKING_RATE = 1.2
+
+class GoogleSynthesizer(BaseSynthesizer):
+    def __init__(
+        self, 
+        language_code: str = DEFAULT_GOOGLE_LANGUAGE_CODE,
+        voice_name: str = DEFAULT_GOOGLE_VOICE_NAME,
+        pitch: int = DEFAULT_GOOGLE_PITCH,
+        speaking_rate: int = DEFAULT_GOOGLE_SPEAKING_RATE,
+        ):
+        self.tts = tts
+        
+        # Instantiates a client
+        credentials_path = getenv("GOOGLE_APPLICATION_CREDENTIALS")
+        if not credentials_path:
+            raise Exception(
+                "Please set GOOGLE_APPLICATION_CREDENTIALS environment variable"
+            )
+        os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = credentials_path
+        self.client = tts.TextToSpeechClient()
+
+        self.voice = tts.VoiceSelectionParams(
+            language_code=DEFAULT_GOOGLE_LANGUAGE_CODE,
+            name=DEFAULT_GOOGLE_VOICE_NAME,
+        )
+
+        self.audio_config = tts.AudioConfig(
+            audio_encoding=tts.AudioEncoding.LINEAR16,
+            sample_rate_hertz=24000,
+            speaking_rate=DEFAULT_GOOGLE_SPEAKING_RATE,
+            pitch=DEFAULT_GOOGLE_PITCH,
+            effects_profile_id=["telephony-class-application"],
```

Comment:
q: what does this line do / should it be set to telephony in all use cases?

Reply:
based on this seems like it should only be set to telephony when it's being used as such 

https://cloud.google.com/text-to-speech/docs/audio-profiles